### DISCUSSION TITLE
what type of chat_id using for bot.send_message()?

### DISCUSSION THREAD

**Discussion by mohammadali-seifkashani** [kianahmadian/python-bale-bot#26] [Q&A] [answered]
سلام و عرض ادب. 
یه نکته ای که حدود نیم ساعت وقت بنده رو گرفت، این بود که در bot.send_message() که در documentation گذاشته شده، نگفتید که کدام chat_id را باید استفاده کرد؟
message.chat_id is in this format: csis_crawler_bot#767300779
client.user.chat_id is in this format: 767300779
ابتدا فکر میکردم که باید chat_id را از طریق user دسترسی داشته باشم و معقولش هم همین است. حتی معقولش این است که خروجی هر دو پارامتر بالا یکی باشد در حالی که نیست!
خلاصه فهمیدم که به جای client.user.chat_id باید از message.chat_id استفاده کرد.
لطفا documentation را (و نیز کد را) ویرایش کنید. 
با تشکر.

**Comment by kian-ahmadian** [comment, marked as answer]
وقت بخیر

### دو نکته، **در داکیومنت ارائه شده کاملا گفته شده چت آیدی مورد نظر باید از نوع استرینگ و یا عدد باشد.**

### **همچنین در صورتی که در جواب پیامی میخواهید اتفاقی رخ دهد بایستی از طریق کلاس همان پیام به متد های مربوطه برسید.** -> یعنی با استفاده از ایونت ها پیش برید. مثال ها در دایرکتوری [به همین نام](https://github.com/python-bale-bot/python-bale-bot/tree/master/examples) داده شده است.

- به [سایت پروژه](https://python-bale-bot.ir) هم میتونین مراجعه کنین. 

در ضمن حتی در صورتی که با دقت کد رو بررسی میکردین متوجه میشدید که در حال ارسال پیام از طرف ربات به خود ربات هستید!

در صورتی که بازم نکته ای بود، برای راهنمایی بیشتر بفرمایید.
